src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -0,0 +1,334 @@
+using System;
+using System.IO;
+using System.Text;
+using System.Security;
+using System.Management.Automation;
+using System.Collections.Generic;
+using System.Management.Automation.Internal;
+// Once Serialization is available on CoreCLR: using System.Runtime.Serialization.Formatters.Binary;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Displays the hexidecimal equivalent of the input data.
+    /// <summary>
+    [Cmdlet(VerbsCommon.Format, "Hex", SupportsShouldProcess = true, HelpUri ="https://go.microsoft.com/fwlink/?LinkId=526919")]
+    [Alias ("fhx")]
+    [OutputType(typeof(Microsoft.PowerShell.Commands.ByteCollection))]
+    public sealed class FormatHex : PSCmdlet
+    {
+        private const int BUFFERSIZE = 16;
+
+        #region Parameters
```

Comment:
Is there any chance we could get `Count` and `Offset` parameters?  I could probably live without `Offset` but on the PSCX Format-Hex command, I use `-Count` almost all the time.  That's because I use it to see if a text file has a BOM, in which case I only want to see the first 3 characters.